neuralpp/test/quick_tests/symbolic/z3_usage_test.py

```diff
@@ -258,3 +258,20 @@ def test_z3_fp_sort():
     assert fp_add.domain(1) == double_sort
     assert fp_add.domain(2) == double_sort
     assert fp_add.range() == double_sort
+
+
+def test_z3_simplify():
+    x = Int('x')
+    g = Goal()
+    g.add(x > 1)
+    g.add(x < 3)
+    r = Tactic('simplify')(g)
+    r = simplify(r.as_expr())
+    # limited, cannot get x == 2
+    assert repr(r) == "And(Not(x <= 1), Not(3 <= x))"
```

Comment:
I thought this should be  `"And(Not(x <= 1), Not(3 >= x))"`, since `x < 3` becomes `Not(x >= 3)` and not `Not(x <= 3)`.  Does this test pass?

Reply:
@CactusWin isn't `3 <= x` equivalent to `x >= 3` ;)?

Reply:
Ignore this! I can't read this morning